libraries/aws_iam_password_policy.rb

```diff
@@ -68,4 +68,8 @@ def number_of_passwords_to_remember
       unless prevents_password_reuse?
     @policy.password_reuse_prevention
   end
+
+  def to_s
+    "IAM Password-Policy"
```

Comment:
We should use single-quoted string here as there is no string interpolation happening.